### ISSUE TITLE
Train/Val Split in DAIR‑V2X‑I

### ISSUE THREAD

**Issue by sucreil97** [zhangzhang2024/PillarID#1]
May I ask whether you further split a validation set from the original training set of DAIR‑V2X‑I, and used the original validation set as the test set?

**Comment by zhangzhang2024** [comment]
Thanks for your question.

We used the original training set of DAIR-V2X-I for model training and did not further split it into an additional validation set. During training, we simply followed the full training split as provided in the dataset.

For evaluation, we used the model checkpoint from the last training epoch as the final model weights. The original validation set provided by DAIR-V2X-I was used directly as the evaluation (test) set for reporting results.
